.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 pg_es_fdw.egg-info/
 .make
+.vscode/
```

Comment:
This is a change specific to your editor that should be handled using a global gitignore: https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
as 

> Thank you for submitting this PR. I've added a few comments where things could be improved.
> 
> I am happy to pick this work up at this point if you wish, or you can address the comments and then I will review again.

I can work on this and resubmit. Thanks for reviewing it.